src/components/organisms/datadl.vue

```diff
@@ -56,6 +58,17 @@
       isUri(object) {
        return object.type === 'uri'
       },
+      getDataUri(triple) {
+        let uri = this.isSubject(triple.o) ? triple.s.value : triple.o.value
+        if (uri.indexOf('.gent/id/') !== -1) {
```

Comment:
Suggestion: Don't know if it's really necessary to have this `if`-statement, since the replace simply won't do anything if `.gent/id/` isn't found in the string.

Instead you could just try something like:
```javascript
getDataUri(triple) {
  let uri = this.isSubject(triple.o) ? triple.s.value : triple.o.value
  return uri.replace('.gent/id/', '.gent/data/')
},

```

Reply:
:+1: